cmd/setup.go

```diff
@@ -163,7 +164,7 @@ func newSetupCommand(config *settings.Config) *cobra.Command {
 
 func setup(opts setupOptions) error {
 	if shouldAskForToken(opts.cfg.Token, opts.tty) {
-		token, err := opts.tty.readTokenFromUser("CircleCI API Token")
+		token, err := opts.tty.readTokenFromUser("CircleCI API Token (which can be created here: https://circleci.com/account/api)")
```

Comment:
It definitely makes sense to include instructions for generating an API token in the README of the project.

As a general rule of thumb, we try not to hard code links throughout our projects as they are subject to change.